pkg/admission/validator/shoot_test.go

```diff
@@ -300,11 +300,15 @@ var _ = Describe("Shoot validator", func() {
 				c.EXPECT().Get(ctx, cloudProfileKey, &gardencorev1beta1.CloudProfile{}).SetArg(2, *cloudProfile)
 
 				shoot.Spec.Networking.Nodes = nil
+				shoot.Spec.Networking.IPFamilies = []core.IPFamily{core.IPFamilyIPv4, core.IPFamilyIPv6}
```

Comment:
Can you add a positive test case for allowing single-stack configurations?

Reply:
Added single-stack IPv6-only test in https://github.com/gardener/gardener-extension-provider-aws/pull/1095/commits/8894dd503369c57173b9112575fef25b6e092268.

PTAL.